app/wordpress/[slug]/page.tsx

```diff
@@ -12,6 +12,9 @@ import formatBlockData from "@/functions/gutenberg/formatBlockData";
 // Import component dependencies
 import Blocks from "@/components/gutenberg/Blocks/Blocks";
 
+// Import TypeScript definitions
+import { GutenbergBlockProps } from "@/types/gutenberg";
+
```

Comment:
@Danishkhurshid Did you look into moving this under the block types we've defined as discussed?